### PR TITLE
OTWO-4908 Add CRUD actions for ActiveAdmin

### PR DESCRIPTION
**Pull request by drubio1989**

### CODE REVIEW

**Comment by drubio1989** [comment]
retest this please

File: test/integration/admin/license_permission_role_admin_test.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    assert_response :success
+  end
+
+  it 'should load the edit page' do
+    license_permission_role = create(:license_permission_role)
+    get edit_admin_license_permission_role_path(license_permission_role)
+    assert_response :success
+  end
+
+  it 'should be able to update a license permission role' do
```
Comment (msk999) [inline test/integration/admin/license_permission_role_admin_test.rb]: Are you allowing a license role here or being forbidden?

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Pull request merged**